### DISCUSSION TITLE
Generating Certificate Signing Request (CSR) code

### DISCUSSION THREAD

**Discussion by Babatunde-Fatai** [vercel/community#1496] [Help] [answered]
### Summary

Hi, I got an SSL certificate from a third party and they are requesting CSR code generated by Vercel from me because my domain is currently pointed/hosted on Vercel, how do I go about generating the code?

**Comment by swarnava** [comment, marked as answer]
Hi there,

We don't provide CSR to generate SSL certificate as our certificate generation is automatically happen in our server. Custom SSL certificates are only available under the Enterprise plan
You can see the details here:
https://vercel.com/pricing
